src/tds/numeric.rs

```diff
@@ -294,8 +300,14 @@ mod bigdecimal_ {
     into_sql!(self_,
             BigDecimal: (ColumnData::Numeric, {
                 let (int, exp) = self_.as_bigint_and_exponent();
-                let value = int.to_i128().expect("Given BigDecimal overflowing the maximum accepted value.");
-                let scale = u8::try_from(exp).expect("Given exponent overflowing the maximum accepted scale (255).");
+                let mut value = int.to_i128().expect("Given BigDecimal overflowing the maximum accepted value.");
+
+                if exp < 0 {
+                    value *= i128::from(10i64.pow(u32::try_from(exp.abs()).expect("Given BigDecimal exponent underflowing the minimum accepted scale (-9223372036854775808).")));
```

Comment:
A comment here would be nice. Suggestion: "The decimal has a negative exponent. We multiple by 10^i where i is the exponent's negative value in order to get a decimal that's of the form a * 10⁰.".